Sources/BraintreeThreeDSecure/Public/BraintreeThreeDSecure/BTThreeDSecureV2ButtonCustomization.h

```diff
@@ -0,0 +1,28 @@
+#import <Foundation/Foundation.h>
+
+#if __has_include(<Braintree/BraintreeThreeDSecure.h>)
+#import <Braintree/BTThreeDSecureV2BaseCustomization.h>
+#else
+#import <BraintreeThreeDSecure/BTThreeDSecureV2BaseCustomization.h>
+#endif
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * The ButtonCustomization class provides methods for the 3DS Requestor App to pass button customization parameters to the 3DS SDK.
+ */
+@interface BTThreeDSecureV2ButtonCustomization : BTThreeDSecureV2BaseCustomization
+
+/**
+ * @property backgroundColor Colour code in Hex format. For example, the colour code can be “#999999”.
```

Comment:
```suggestion
 * @property backgroundColor Color code in Hex format. For example, the color code can be “#999999”.
```

Not sure if these came from Cardinal docs and we want them to match, but could change this to match the variable name.

Reply:
Good call!

Reply:
I went through and updated a bunch of these doc strings to be more clear and correct. I don't think they need to match Cardinal's docs exactly.